root/layout/components/sidebar/SidebarAcousticBrainz.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React, {useEffect, useState} from 'react';
```

Comment:
Please use `import * as React` (I'm changing the place you copied this one from in https://github.com/metabrainz/musicbrainz-server/pull/1580/files) for this one :)